lib/ansible/module_utils/nxos.py

```diff
@@ -288,18 +288,19 @@ def send_request(self, commands, output='text', check_status=True):
             except ValueError:
                 self._module.fail_json(msg='unable to parse response')
 
-            output = response['ins_api']['outputs']['output']
-            for item in to_list(output):
-                if check_status and item['code'] != '200':
-                    self._error(output=output, **item)
-                elif 'body' in item:
-                    result.append(item['body'])
-                # else:
-                    # error in command but since check_status is disabled
-                    # silently drop it.
-                    # result.append(item['msg'])
-
-        return result
+            if 'outputs' in response.keys():
```

Comment:
This shouldn't be in `response.keys()`, it's in `response['ins_api'].keys()`. I would rather `if response['ins_api'].get('outputs'):`